src/edu/stanford/dlss/was/WasapiCrawlSelector.java

```diff
@@ -7,10 +7,15 @@
 
 public class WasapiCrawlSelector {
 
-  public Map<Integer, List<WasapiFile>> crawlIdToFiles;
+  protected Map<Integer, List<WasapiFile>> crawlIdToFiles = new HashMap<Integer, List<WasapiFile>>();
```

Comment:
i had vague thoughts about leaving things open for extension... seems like the sort of thing a subclass might want access to?  i don't feel strongly about it.